docs/reference/module-overview.md

```diff
@@ -54,13 +54,11 @@ modules.export = {
 };
 ```
 
-You should not use both `improve` and `extend` in a single module.
-
 ### `improve`
 
 Similarly to `extend`, `improve` is used to name another existing module. Instead of _adopting_ the existing module's configuration, using `improve` _changes_ the named module's configuration.
 
-**This is only valid in modules that are built independently and then installed into Apostrophe apps**, either on their own or as a part of a bundle. It is most often used to improve core Apostrophe modules.
+**This is only valid in modules that are installed into an Apostrophe app**, either on their own or as a part of a bundle, and not in those that are built into the app directly. It is most often used to add functionality to core Apostrophe modules.
```

Comment:
If you need to alter the behavior of an installed module in your application-level code, just create an `index.js` file at project level and put your additional code and configuration there. `improve` does much the same thing, but allows you to do it in an npm module where the name can't be the same as the original.

Reply:
^^^ I think the above is important.

Reply:
I added that in a note right after this.